pb/summary/summary.proto

```diff
@@ -178,6 +178,7 @@ message DashboardTabSummary {
     STALE = 5;
     BROKEN = 6;
     PENDING = 7;
+    ACCEPTABLE = 8;
```

Comment:
You can also mark the `acceptably_flaky` field as deprecated if you don't intend it to be used going forward. (similar to some other fields in this proto).